tests/specs/global.spec.js

```diff
@@ -1,87 +1,86 @@
-import initialize, { app, modalPage } from "../common.js";
+import initialize, { app, portfolioPage, modalPage } from "../common.js";
 
 describe("Global", () => {
   initialize("global", {
     userData: "1AccountBTC1AccountETHStarred",
   });
 
-  const $ = selector => app.client.$(selector);
+  beforeEach(async () => {
+    const drawerPortfolioButton = await portfolioPage.drawerPortfolioButton;
+    await drawerPortfolioButton.click();
+  });
 
   it("can open send modal", async () => {
-    const sendButton = await $("#drawer-send-button");
+    const sendButton = await portfolioPage.drawerSendButton;
     await sendButton.click();
     await modalPage.isDisplayed();
     expect(await app.client.screenshot(2000)).toMatchImageSnapshot({
       customSnapshotIdentifier: "global-send-modal",
     });
+    await modalPage.close();
   });
 
   it("can open receive modal", async () => {
-    await modalPage.close();
-    const sendButton = await $("#drawer-receive-button");
-    await sendButton.click();
+    const receiveButton = await portfolioPage.drawerReceiveButton;
+    await receiveButton.click();
     await modalPage.isDisplayed();
     expect(await app.client.screenshot()).toMatchImageSnapshot({
       customSnapshotIdentifier: "global-receive-modal",
     });
+    await modalPage.close();
```

Comment:
expect needs to be the last instruction of an it.
otherwise it breaks what comes next and it can be annoying (next tests and screenshot will be more wrong than they should)